apps/web/app/s/[surveyId]/page.tsx

```diff
@@ -25,8 +27,54 @@ interface LinkSurveyPageProps {
   };
 }
 
+export async function generateMetadata({ params }: LinkSurveyPageProps): Promise<Metadata> {
+  const survey = await getSurvey(params.surveyId);
+
+  if (!survey || survey.type !== "link" || survey.status === "draft") {
+    notFound();
+  }
+
+  const product = await getProductByEnvironmentId(survey.environmentId);
+
+  if (!product) {
+    throw new Error("Product not found");
+  }
+
+  function getNameForURL(string) {
+    return string.replace(/ /g, "%20");
+  }
+
+  function getBrandColorForURL(string) {
+    return string.replace(/#/g, "%23");
+  }
+
+  const brandColor = getBrandColorForURL(product.brandColor);
+  const surveyName = getNameForURL(survey.name);
+
+  const ogImgURL = `${WEBAPP_URL}/api/v1/environments/${survey.environmentId}/surveys/${survey.id}/og?brandColor=${brandColor}&name=${surveyName}`;
+
+  return {
+    openGraph: {
+      title: "Formbricks",
```

Comment:
Same here, please fill dynamically

Reply:
Sure, I thought the title here was the website name and the dynamic one is only inside the image, small change - will do it soon